source/gfoidl.Base64/Internal/Base64Encoder/Base64Encoder.Decode.netcoreapp2.1.cs

```diff
@@ -61,7 +61,7 @@ partial class Base64Encoder
                 maxSrcLength = (destLength / 3) * 4;
             }
 
-            ref sbyte decodingMap = ref s_decodingMap[0];
+            ref sbyte decodingMap = ref MemoryMarshal.GetReference(DecodingMap.Slice(1));
```

Comment:
Add comment to https://github.com/dotnet/coreclr/issues/23194 here. Other places as well.